File: src/main/scala/assignment3/e0/ParticleSystemWithGUI.java
package assignment3.e0;

/**
 * PCD 2018-2019 - Assignment #01
 * 
 * Particle system version with GUI
 * 
 * @author aricci
 *
 */
public class ParticleSystemWithGUI {
	public static void main(String[] args) {

		double dt = 0.01;		
		int windowSizeX = 1200;
		int windowSizeY = 1000;
		double scaleFactor = 10;
		boolean displayAllSnapshot = false;
		
		World world = new World(dt, displayAllSnapshot);		
		WorldViewer viewer = new WorldViewer(world, windowSizeX, windowSizeY, scaleFactor);
		Controller controller = new Controller(world, viewer);
		viewer.setController(controller);		
		viewer.show();
	}
}

File: src/main/scala/assignment3/e0/ParticleSystemWithGUI.scala
package assignment3.e0

object ParticleSystemWithGUI extends App {

  override def main(args: Array[String]): Unit = {
    val dt = 0.01
    val windowSizeX = 1200
    val windowSizeY = 1000
    val scaleFactor = 10

    val world = new World(dt)
    val viewer = new WorldViewer(world, windowSizeX, windowSizeY, scaleFactor)
    val controller = new Controller(world, viewer)
    viewer.setController(controller)
    viewer.show()
  }

}